app/pages/index.vue

```diff
@@ -42,6 +42,7 @@ useSeoMeta({
   <main class="grow w-full lg:w-2/3 mx-auto mt-20 p-2">
     <SiteHeader />
     <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">Open Positions</h3>
+    <span class="text-sm text-zinc-500">({{ totalPositions }})</span>
```

Comment:
```suggestion
 <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">Open Positions {{totalPositions }}</h3>
```
I thought is that wrap this in the same line to keep align.

again cc: @amandesai01

Reply:
Resolved